IAmALamp/src/components/BottomBar/Button.tsx

```diff
@@ -0,0 +1,29 @@
+import React from "react";
+import { View, Image, TouchableOpacity, Text } from "react-native";
+
+import styles from "./styles";
```

Comment:
Install the prettier plugin on your editor to auto fix these, we are using single quotes